tests/integration_tests/test_botorch.py

```diff
@@ -0,0 +1,249 @@
+from typing import cast
+from typing import Optional
+from typing import Sequence
+from unittest.mock import patch
+
+import pytest
+import torch
+
+from optuna import integration
+from optuna import multi_objective
+from optuna.integration import BoTorchSampler
+from optuna.samplers import RandomSampler
+from optuna.storages import RDBStorage
+from optuna.trial import FrozenTrial
+from optuna.trial import Trial
+
+
+@pytest.mark.parametrize("n_objectives", [1, 2, 4])
+def test_botorch_candidates_func_none(n_objectives: int) -> None:
+    n_trials = 3
+    n_startup_trials = 2
+
+    sampler = BoTorchSampler(n_startup_trials=n_startup_trials)
+
+    study = multi_objective.create_study(["minimize"] * n_objectives, sampler=sampler)
```

Comment:
How about replacing the `multi_objective.create_study` with `optuna.create_study`? #1994 has been merged and you update the BoTorch example with it.
```suggestion
    study = optuna.create_study(directions=["minimize"] * n_objectives, sampler=sampler)
```

Reply:
Thanks, I should've noticed that. Fixed!